docs/mixin/alerts/alerts.yaml

```diff
@@ -189,4 +189,90 @@ groups:
       severity: warning
     annotations:
       summary: High cache eviction in Promscale
-      description: "Promscale {{ $labels.name }} has a hit ratio of {{ $value }}."
\ No newline at end of file
+      description: "Promscale {{ $labels.name }} has a hit ratio of {{ $value | humanizePercentage }}."
+- name: promscale-database
+  rules:
+  - alert: PromscaleStorageUnhealthy
+    expr: |
+      (
+        sum by (job) (
+          rate(promscale_database_health_check_errors_total[5m])
+        )
+      /
+        sum by (job) (
+          rate(promscale_database_health_checks_total[5m])
+        )
+      ) > 0.05
+    labels:
+      severity: warning
+    annotations:
+      summary: Promscale unhealthy storage
+      description: "Promscale connection with the database has an error of {{ $value | humanizePercentage }}."
+  - alert: PromscaleStorageUnhealthy
+    expr: |
+      (
+        sum by (job) (
+          rate(promscale_database_health_check_errors_total[5m])
+        )
+      /
+        sum by (job) (
+          rate(promscale_database_health_checks_total[5m])
+        )
+      ) > 0.1
+    labels:
+      severity: critical
+    annotations:
+      summary: Promscale unhealthy storage
+      description: "Promscale connection with the database has an error of {{ $value | humanizePercentage }}."
+  - alert: PromscaleStorageHighErrorRate
+    expr: |
+      (
+        sum by (job) (
+          rate(promscale_database_request_errors_total{method=~"query.*|exec"}[5m]) # Error counter exists for query, query_row & exec, and not for send_batch.
+        )
+      /
+        sum by (job) (
+          rate(promscale_database_requests_total{method=~"query.*|exec"}[5m])
+        )
+      ) > 0.05
+    labels:
+      severity: critical
+    annotations:
+      summary: High error rate in Promscale database connection
+      description: "Promscale connection with the database has an error of {{ $value | humanizePercentage }}."
+  - alert: PromscaleStorageHighLatency
+    expr: |
+      (
+        histogram_quantile(0.99,
+          sum by (le, job, type) (
+            rate(promscale_database_requests_duration_bucket[5m])
+          )
+        ) > 0.5
+      and
+        sum by (job, type) (
+          rate(promscale_database_requests_duration_count[5m])
+        ) > 0
+      )
+    labels:
+      severity: warning
+    annotations:
+      summary: Slow database response
+      description: "99% of Database is taking {{ $value }} seconds to respond to Promscale's requests."
+  - alert: PromscaleStorageHighLatency
+    expr: |
+      (
+        histogram_quantile(0.99,
+          sum by (le, job, type) (
+            rate(promscale_database_requests_duration_bucket[5m])
+          )
+        ) > 3
+      and
+        sum by (job, type) (
+          rate(promscale_database_requests_duration_count[5m])
+        ) > 0
+      )
+    labels:
+      severity: critical
```

Comment:
Database connection by itself is not directly affecting users, but it can affect them indirectly. As such it is not a symptom, but a cause and shouldn't have critical severity.

Alerts with critical severity should be limited **only** to components that are affecting user **directly**, such as data ingest and data query.

More information can be found in [My Philosophy on Alerting](https://docs.google.com/a/boxever.com/document/d/199PqyG3UsyXlwieHaqbGiWVa8eMWi8zzAn0YfcApr8Q/edit) based on Rob Ewaschuk's observations at Google.